src/witness/operation.rs

```diff
@@ -829,35 +832,43 @@ pub(crate) fn load_preimage<F: Field>(
 
     let mut cpu_row = CpuColumnsView::default();
```

Comment:
As discussed in DM,  we move the preimage hash check back to Minigeth, so we can remove code from line 833 to 866